src/drag.js

```diff
@@ -63,12 +63,20 @@ var elementFromPoint = function (point, element) {
 		var el = elementFromPoint(point, element);
 		if (last !== el && el && last) {
 			var options = syn.helpers.extend({}, point);
+
+			// QUESTION: Should we also be sending a pointerleave event?
 			options.relatedTarget = el;
+			if(syn.support.pointerEvents){syn.trigger(last, 'pointerout', options);}
 			syn.trigger(last, "mouseout", options);
+
+			// QUESTION: Should we also be sending a pointerenter event?
```

Comment:
Can these questions be answered by something in the specs, or how else can we get them answered?